project-icp/bee-pollinator-app
# Skip Analyze

## Overview

After a shape is drawn, and the user clicks "Confirm", they are taken to the modeling view instead of analyze. Analyze is completely removed from the front-end: it no longer exists as its own page or as a tab in the modeling view. The back button in the modeling view has also been removed.

### Demo

![bee-transition-to-model](https://cloud.githubusercontent.com/assets/1430060/20189090/a8e14458-a749-11e6-99d9-3c3f18d81dee.gif)

### Notes

The modeling view currently fires TR-55, which was easier than setting up a dummy endpoint.

I've created #32 and #33 for issues discovered, but not addressed, while working on this.

## Testing Instructions

Pull down the branch. Run the bundle script:

```
./scripts/bundle.sh --debug --vendor
```

Load [http://localhost:8000](http://localhost:8000). Draw a shape, and click "Confirm". You should be taken to the modeling view with the correct shape loaded. The actual model may not run (for uninvestigated reasons).

Builds on top of #29 
Connects #23
